### ISSUE TITLE
Blade of Agony "classic" statusbar support

### ISSUE THREAD

**Issue by Riccardoman** [3saster/fullscrn_huds#5]
It would be nice to have support for the new release of BoA ch 3

**Comment by NightFright2k19** [comment]
That statusbar differs from the regular one quite a bit. Might not be impossible to pull off, but significant code adjustments would be required. I cannot pull this off by myself, I'm afraid. 3saster needs to take a look, but he hasn't been around much lately, so it's probably not going to happen any time soon (if it should).

**Comment by 3saster** [comment]
This sort of addition is out-of-scope for this project I think. That HUD adds quite a lot of new additions afaik, and this statusbar was always meant to be a sort of "vanilla+" kind of HUD. For maps or mods that add significant new features to the HUD, I would recommend using the mod's HUD instead, or making your own mod for it.

As for me not being around a lot, I do want to return and at least finish off what I consider left before this is essentially feature-complete (namely finishing idgames compatibility, adding Strife support, and possibly adding a degree of allowing moving the HUD around). I burned out while doing the idgames compatibility (got about halfway through the idgames archive) and have generally been taking a bit of a break from Doom lately (thanks Binding of Isaac!), but rest assured, I will be returning to finish this off at some point, there is a lot of work I already did that I don't want to go to waste (and Agent_Ash provided me a fantastic widescreen version of Curse of D'Sparil's HUD I would love to use).
